cost/google/idle_compute_instances/google_idle_compute_instances.pt

```diff
@@ -357,9 +360,12 @@ policy "pol_utilization" do
     escalate $terminate_resources
     export do
       resource_level true
-      field "projectId" do
+      field "projectID" do
         label "Project ID"
       end
+      field "projectName" do
+        label "Project Name"
+      end
```

Comment:
Should we add standard `accountID` and `accountName` fields?

Reply:
Should we scrape recommendations from this "Google Idle Compute Instances" policy?

Reply:
Not scraping from this policy until it can provide accurate cost data.